### DISCUSSION TITLE
What should I do with Blitz now ???

### DISCUSSION THREAD

**Discussion by IRediTOTO** [blitz-js/blitz#3091] [Q&A]
Hi, I am trying coding my project (my dream) . Sure, I'm using Blitz. But Blitz will be changed, and this package will be old package, no update anymore... Can you give me an advice to change back to NextJS, no API layer is so awesome, I feel can't live without it...

**Comment by abuuzayr** [comment]
Hi @IRediTOTO , you should just build what you want with Blitz currently. A migration path with be provided once the toolkit is announced. Sure this package will have no updates for now, but there are production apps running on even older Blitz . mine are running on 0.41 i think? No issues. You should be able to build what you want with what we have so far.

**Reply by beerose** [reply]
I want to add to @abuuzayr answer that until the toolkit is ready (and probably sometime after that) we'll be fixing major bugs and releasing patches.

**Reply by IRediTOTO** [reply]
yes, I'd love to continue use Blizt, i hove can migration easy in future. I just want to sure can use Blitz for long time, not remake all project
